setup.py

```diff
@@ -2,36 +2,43 @@
 
 from setuptools import setup, find_packages
 
-install_requires = [
+INSTALL_REQUIRES = [
     'lxml',
     'setuptools',
-    'Django>=1.4',
+    'Django>=1.4,<1.7',
     'django-appconf',
     'django-cms>=2.4',
     'django-classy-tags>=0.3.2',
-    'django-haystack>=2.0.0',
+    'django-haystack>=2.4.0.dev0',
```

Comment:
@czpython told me via hangouts that we should keep a django-haystack branch for this DoNotIndex feature and document it in REAME. If django-haystack #1191 does not backport to 2.3.x I will keep 2.3.x branch with that feature and we use it, else, is ok to just use django-haystack>=2.3.2 after it's release.